instrumentation/aws_sdk/test/opentelemetry/instrumentation_test.rb

```diff
@@ -55,216 +65,4 @@
       instrumentation.instance_variable_set(:@installed, false)
     end
   end
-
-  describe 'validate_spans' do
-    describe 'SNS' do
-      it 'should have correct attributes' do
-        sns = Aws::SNS::Client.new(stub_responses: true)
-        sns.stub_responses(:publish)
-
-        sns.publish message: 'msg'
-
-        _(last_span.attributes['rpc.system']).must_equal 'aws-api'
-        _(last_span.attributes['rpc.service']).must_equal 'SNS'
-        _(last_span.attributes['rpc.method']).must_equal 'Publish'
-        _(last_span.attributes['aws.region']).must_include 'stubbed'
-        _(last_span.attributes['db.system']).must_be_nil
-
-        _(last_span.attributes[OpenTelemetry::SemanticConventions::Trace::HTTP_STATUS_CODE]).must_equal 200
-
-        _(last_span.status.code).must_equal OpenTelemetry::Trace::Status::UNSET
-      end
-    end
-
-    describe 'S3' do
```

Comment:
I don't see a replacement for these S3 tests. Is that intentional?

Reply:
Yes, this was intentional. The S3 gem is rather large (465 KB) so we decided to replace this with Lambda (140KB) to make the testing lightweight. Also - there's no S3-specific span attributes that are being generated so we felt Lambda was sufficient to test the same functionality.

Reply:
Got it, that makes a lot of sense. Thank you!